drivers/gamatronic.c

```diff
@@ -69,8 +69,11 @@ static int sec_upsrecv (char *buf)
 				strncpy(lenbuf, buf+2, 3);
 				lenbuf[3] = '\0';
 				ret = atoi(lenbuf);
+				if (ret > 140) {
+					ret = 140;
```

Comment:
Since the value is hardcoded, would it be good also to include a debug log indicating the possible error? I keep thinking about a manufacturer trying to add NUT support and correcting their own implementation or someone like me who spent a considerable amount of time trying to understand C code and debug the cause of this error without success.

Reply:
I had written a change for this, and when I went to update the PR I saww @jimklimov had beaten me to it! https://github.com/networkupstools/nut/compare/5086b2b..ff045c0